src/account/0.6/PasskeyAccount.sol

```diff
@@ -1,10 +1,13 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.13;
 
-import { Base64Url } from "../utils/Base64Url.sol";
-import { IEntryPoint, UserOperation } from "@aa/interfaces/IEntryPoint.sol";
-import { IPasskeyAccount, Passkey } from "../interfaces/IPasskeyAccount.sol";
-import { SimpleAccount } from "@aa/samples/SimpleAccount.sol";
+import { Base64Url } from "./utils/Base64Url.sol";
+import {
+    IEntryPoint,
+    UserOperation
+} from "@aa/0.6/contracts/interfaces/IEntryPoint.sol";
+import { IPasskeyAccount, Passkey } from "../../interfaces/IPasskeyAccount.sol";
+import { SimpleAccount } from "@aa/0.6/contracts/samples/SimpleAccount.sol";
```

Comment:
I would like to make library import more comprehensive

`@aa` -> `@account-abstraction`
`@oz` -> `@openzeppelin-contracts`

Reply:
OK, the `@openzeppelin` remappings need to be moved down in `foundry.toml` because they will conflict with `lib/account-abstraction/0.6 (or 0.7):@openzeppelin`.

Reply:
Why `@openzeppelin-contracts` would conflict with `lib/account-abstraction/0.6:@openzeppelin`?

Reply:
Sorry, I misunderstood. `@openzeppelin-contracts` doesn't conflict with `@openzeppelin`.